lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -6,15 +6,20 @@
             'gettext',
             'js/views/fields',
             'teams/js/models/team',
+            'common/js/components/utils/view_utils',
             'text!teams/templates/edit-team.underscore'],
-        function (Backbone, _, gettext, FieldViews, TeamModel, editTeamTemplate) {
+        function (Backbone, _, gettext, FieldViews, TeamModel, ViewUtils, editTeamTemplate) {
             return Backbone.View.extend({
 
                 maxTeamNameLength: 255,
                 maxTeamDescriptionLength: 300,
 
+                submitNoDoubleCLick: function(event) {
```

Comment:
Nice catch - I had meant to ask about the placement of that. Since `createOrUpdateTeam` is hard-coded in, I think down near there is fine. Should be fixed now.